doc/conf.py

```diff
@@ -174,26 +174,24 @@
     # Standard Sphinx extensions
     'sphinx.ext.autodoc',
     'sphinx.ext.extlinks',
+	  'sphinx.ext.inheritance_diagram',
     'sphinx.ext.intersphinx',
     'sphinx.ext.todo',
-#    'sphinx.ext.graphviz',
+    'sphinx.ext.graphviz',
     'sphinx.ext.mathjax',
     'sphinx.ext.ifconfig',
     'sphinx.ext.viewcode',
 
     # BuildTheDocs extensions
     'btd.sphinx.autoprogram',
-    'btd.sphinx.graphviz',
-    'btd.sphinx.inheritance_diagram',
+#    'btd.sphinx.graphviz',
```

Comment:
Are BTD forks not required anymore? Shall we archive them?

Reply:
I believe it's still used in multiple places. Some original extensions got updates, others not yet.

The problem was, when modifying extension A, also it's using extensions B, ... need to be modifies, otherwise the modification from A was not used.

But yes, we can reevaluate what's still needed and reduce maintenance affords.